scripts/2017/mypy-annotations.py

```diff
@@ -0,0 +1,81 @@
+     "zerver/webhooks/trello/tests.py", "zerver/webhooks/splunk/view.py", "zerver/webhooks/circleci/view.py"],
+    ["zerver/views/users.py", "zerver/webhooks/librato/view.py", "zerver/tests/test_management_commands.py",
+     "zerver/webhooks/ifttt/view.py", "zerver/webhooks/gitlab/view.py", "zerver/webhooks/github_webhook/view.py",
+     "zerver/webhooks/pingdom/view.py", "zerver/tornado/event_queue.py", "zerver/webhooks/newrelic/view.py"],
+    ["zerver/lib/request.py", "zerver/lib/integrations.py", "zerver/webhooks/codeship/view.py",
+     "zerver/webhooks/deskdotcom/view.py", "zerver/webhooks/teamcity/view.py", "zerver/lib/soft_deactivation.py",
+     "zerver/lib/export.py", "zerver/webhooks/crashlytics/view.py", "zerver/tests/test_events.py"],
+    ["zerver/views/user_settings.py", "zerver/lib/avatar.py", "zerver/views/realm_filters.py",
+     "zerver/tests/test_decorators.py", "zerver/webhooks/travis/view.py", "zerver/tests/test_subdomains.py",
+     "zerver/management/commands/send_password_reset_email.py", "zerver/webhooks/helloworld/view.py",
+     "zerver/webhooks/heroku/view.py", "zerver/webhooks/basecamp/view.py", "zerver/tests/test_docs.py", "zerver/views/realm.py"],
+    ["zerver/webhooks/homeassistant/view.py", "scripts/lib/node_cache.py", "zilencer/views.py",
+     "zerver/webhooks/taiga/view.py", "analytics/management/commands/populate_analytics_db.py",
+     "zerver/webhooks/yo/view.py", "tools/lister.py", "analytics/tests/test_counts.py", "zerver/tests/test_messages.py"],
+    ["zerver/views/messages.py", "zerver/views/auth.py", "zerver/webhooks/gci/view.py"],
+    ["zerver/lib/upload.py", "zerver/webhooks/wordpress/view.py", "zerver/views/streams.py",
+     "zerver/views/custom_profile_fields.py"],
+    ["zerver/lib/events.py", "zerver/webhooks/stripe/view.py", "zerver/webhooks/sentry/view.py",
+     "zerver/forms.py", "zerver/webhooks/appfollow/view.py", "zerver/views/zephyr.py", "zerver/webhooks/gosquared/view.py",
+     "zerver/tests/test_auth_backends.py", "zerver/webhooks/bitbucket2/tests.py"],
+]
+
+description_A = """Help convert mypy annotations for multiline methods in the Zulip codebase!
+This is a great way to practice git, git grep, and efficiently using a powerful editor.
+
+Instructions for mypy annotation task are at
+https://github.com/zulip/zulip-gci/blob/master/tasks/2017/mypy-annotations.md.
+
+For this task, do **Task Type A** for the following set of files:
+
+%s
+"""
+
+for files in tasks_A:
+    file_list = "\n".join([("* " + file_name) for file_name in files]),
+    upload_task(
+        # https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf
+        name = 'Update mypy annotations to Python 3 syntax',
+        description = description_A % file_list,
+        status = 2, # 1: draft, 2: published
+        max_instances = 1,
+        mentors = ['rishig@zulipchat.com', 'tabbott@zulipchat.com'],
+        tags = ['python', 'mypy'], # free text
+        is_beginner = True,
```

Comment:
A few questions, should this be a beginner task and who should be the mentors mentioned here?